app/controllers/users_controller.rb

```diff
@@ -161,7 +168,7 @@ def created_user_redirect_path
   end
 
   def find_resource
-    @user = params[:id] ? User.find_by(id: params[:id]) : current_user
+    @user = User.find_by(id: params[:id]) || User.find_by(id: params[:user_id]) || current_user
```

Comment:
Should we search by :user_id before we do :id? Is there ever a scenario where :id would be passed as organisation id?

Reply:
Alternatively maybe specify by action explicitly if we know when we're expecting each

Reply:
Changed the order